docs/source/introduction_guide.rst

```diff
@@ -314,7 +314,7 @@ An alternative to using a DataModule is to defer initialization of the models mo
             # step is either 'fit' or 'test' 90% of the time not relevant
             data = load_data()
             num_classes = data.classes
-            self.l1 = nn.Linear(..., num_classes)
+            self.l1 = torch.nn.Linear(..., num_classes)
```

Comment:
can we keep it nn?
most pytorch examples also do import torch.nn as nn
we should just do the same at the top in the setup block

Reply:
done for all...